docs/main/user-guides/migration/legacy-storage-to-new-storage/device-to-lvm.md

```diff
@@ -0,0 +1,148 @@
+---
+id: device-to-lvm
+title: Migration from OpenEBS Local PV Device to OpenEBS Local PV LVM
+keywords:
+ - Migration from OpenEBS Local PV Device to OpenEBS LVM Local PV
+ - Local PV Device to Local PV LVM
+ - Local PV Device to Local PV ZFS
+ - Local PV Rawfile to Local PV LVM
+ - Local PV Rawfile to Local PV ZFS
+description: This section outlines the process of migrating OpenEBS Local PV Device to OpenEBS LVM Local PV.
+---
+
+:::info
+The following steps are an example about migrating from legacy storage to new storage.
```

Comment:
'new storage' seems incorrect, need a better phrase.

Reply:
As discussed, we are using the phrase "latest storage solution"